{"name": "Checking credential error message", "status": "passed", "attachments": [{"name": "Meditab_credential_error", "source": "2bdbc4b9-fde6-4755-b090-92ba64dceb34-attachment.png", "type": "image/png"}, {"name": "log", "source": "1972d051-6c04-472d-b08c-0746603a4096-attachment.txt", "type": "text/plain"}], "start": 1691866376590, "stop": 1691866385254, "uuid": "dc01bb6f-0b92-4213-90b8-3113db7004d8", "historyId": "33efcd12ee04a415613ee549a8549421", "testCaseId": "33efcd12ee04a415613ee549a8549421", "fullName": "TestCases.test_01_Login_page.Test_Meditab#test_login_01", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "This is a test case for testing of alert message while entering wrong credentials"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_01_Login_page"}, {"name": "subSuite", "value": "Test_Meditab"}, {"name": "host", "value": "LAPTOP-NIATUPE5"}, {"name": "thread", "value": "24140-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_01_Login_page"}], "links": [{"type": "link", "url": "Meditab - https://iemodemoindia.meditab.com", "name": "Meditab - https://iemodemoindia.meditab.com"}]}